docs/api.md

```diff
@@ -219,6 +219,7 @@
   * [elementHandle.getProperties()](#elementhandlegetproperties)
   * [elementHandle.getProperty(propertyName)](#elementhandlegetpropertypropertyname)
   * [elementHandle.hover()](#elementhandlehover)
+  * [elementHandle.isDisplayed()](#elementhandleisdisplayed)
```

Comment:
`elementHandle.isVisible()` is better since it aligns with `waitForSelector`'s "visible" option.

Reply:
Ok, I will rename to `isVisible`